README.md

```diff
@@ -1 +1,40 @@
-# tapyrus-api-client-examples
\ No newline at end of file
+# tapyrus-api-client-examples
+
+This repository is a sample code of client library generated from Tapyrus API's swagger.yml using [OpenAPI Generator](https://github.com/OpenAPITools/openapi-generator).
+
+We support the following five programming languages.
+  - javascript
+  - ruby
+  - Java
+  - python
+  - swift
+
+## How to generate a client
+
+This document explains how to generate the client code using Docker.
+Detailed installation instructions can be found in the [OpenAPI Generator Github repository](https://github.com/OpenAPITools/openapi-generator#16---docker).
+
+
+To generate the code using Docker, use the [OpenAPI Generator CLI Docker Image](https://hub.docker.com/r/openapitools/openapi-generator-cli/).
+
+You can generate the client code by specifying the following options and executing generate.
+ - `-i`: path to the swagger.yml of tapyrus-api
+ - `-g`: The programming language of the client code to generate.
+ - `-o`: The directory path to output to.
+
+The path of swagger.yml can be a local directory path or a global URL.
```

Comment:
I think these steps should have below.

* clone tapyrus-api repository
* cd into the tapyus-api dir
* run the docker run command in the tapyrus-api dir